uptane/common.py

```diff
@@ -214,8 +214,10 @@ def sign_over_metadata(
   <Exceptions>
     tuf.FormatError, if 'key_dict' is improperly formatted.
 
-    tuf.UnsupportedLibraryError, if an unsupported or unavailable library is
-    detected.
+    tuf.UnsupportedLibraryError, if an unsupported or unavailable cryptography
+    library is chosen.
+
+    uptane.Error, if the given metadata format is not 'json' or 'der'
```

Comment:
You also raise `uptane.Error` if an unsupported keytype is specified in metadata.  See line 113.